src/components/Events/CreateEventContainer/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { createEvent } from '../../../actions/events/events-actions';
+import EventForm from '../EventForm/index';
+
+class CreateEventContainer extends Component {
+
+  state = {
+    title: '',
+    photo: '',
+    description: '',
+    startdate: '',
+    enddate: '',
+    locationname: '',
+    address1: '',
+    address2: '',
+    zipcode: '',
+    city: '',
+    country: '',
+    geolocation: '',
+    eventurl: ''
+  }
+
+  onChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  onSubmit = (event) => {
+    event.preventDefault()
+    this.props.createEvent(this.state)
+    this.setState({
+      title: '',
+      photo: '',
+      description: '',
+      startdate: '',
+      enddate: '',
+      locationname: '',
+      address1: '',
+      address2: '',
+      zipcode: '',
+      city: '',
+      country: '',
+      geolocation: '',
+      eventurl: ''
```

Comment:
what about defining all this in a const at the top, before the class definition and then doing 
```
state = emptyForm

// and here

this.setState(emptyForm)
```